### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by rhcu**
Issue #15

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,3 +2,12 @@
 [![Build Status](https://travis-ci.org/marco-c/coverage-crawler.svg?branch=master)](https://travis-ci.org/marco-c/coverage-crawler)
 
 A crawler to find websites that exercise code in Firefox that is not covered by unit tests
+
+## Usage on local machine
+
+- Create and activate Python 3.6 virtual environment
+- In the virtual environment install requirements for the project running `pip install -r requirements`
+- To install testing utilities run `pip install -r test-requirements.txt`
```
Comment (marco-c) [inline README.md]: Creating a virtual env is not required, it should be the choice of the user.
Let's just say that the requirements need to be installed (`requirements.txt` if you only want to run the project, `test-requirements.txt` if you want to develop).

File: README.md [hunk 2]
```diff
@@ -2,3 +2,12 @@
...
+- If you want to test Python file with Flake8 run `flake8 name_or_file.py`
```
Comment (marco-c) [inline README.md]: Let's remove this, it's unrelated with the usage of the crawler.

File: README.md [hunk 3]
```diff
@@ -2,3 +2,12 @@
 [![Build Status](https://travis-ci.org/marco-c/coverage-crawler.svg?branch=master)](https://travis-ci.org/marco-c/coverage-crawler)
 
 A crawler to find websites that exercise code in Firefox that is not covered by unit tests
+
+## Usage on local machine
```
Comment (marco-c) [inline README.md]: Let's call the paragraph `Usage`

File: README.md [hunk 4]
```diff
@@ -2,3 +2,12 @@
...
+
+- Create and activate Python 3.6 virtual environment
+- In the virtual environment install requirements for the project running `pip install -r requirements`
+- To install testing utilities run `pip install -r test-requirements.txt`
+- If you want to test Python file with Flake8 run `flake8 name_or_file.py`
+- run `latest_cov_build.py` to download the latest coverage build of Firefox
```
Comment (marco-c) [inline README.md]: Replace `run` with `Run` and `latest coverage build of Firefox` with `latest Firefox coverage build`.

File: README.md [hunk 5]
```diff
@@ -2,3 +2,12 @@
...
+- run `crawler.py`
```
Comment (marco-c) [inline README.md]: "Run the `crawler.py` script"

**Comment by marco-c** [comment]
Additional comments:
1) End each line of the list with a `;` and the last one with a `.`;
2) Please merge from master, as the PR contains several unrelated commits.

File: README.md (hunk 5 above)
Comment (rhcu) [reply README.md]: Done. :)

File: README.md [hunk 6]
```diff
@@ -2,3 +2,10 @@
 [![Build Status](https://travis-ci.org/marco-c/coverage-crawler.svg?branch=master)](https://travis-ci.org/marco-c/coverage-crawler)
 
 A crawler to find websites that exercise code in Firefox that is not covered by unit tests
+
+## Usage
+
+- Install requirements for the project running `pip install -r requirements` command if you only want to run project;
+- Install testing utilities requirements with `pip install -r test-requirements.txt` command if you want to develop;
```
Comment (marco-c) [inline README.md]: "Install development requirements with `pip install -r test-requirements.txt`;"

File: README.md [hunk 7]
```diff
@@ -2,3 +2,10 @@
 [![Build Status](https://travis-ci.org/marco-c/coverage-crawler.svg?branch=master)](https://travis-ci.org/marco-c/coverage-crawler)
 
 A crawler to find websites that exercise code in Firefox that is not covered by unit tests
+
+## Usage
+
+- Install requirements for the project running `pip install -r requirements` command if you only want to run project;
```
Comment (marco-c) [inline README.md]: "Install requirements with `pip install -r requirements`;"

**Comment by marco-c** [comment]
Also please rebase on top of master, as there are too many unrelated commits.

**Pull request closed without merging**